src/ru/skypro/Main.java

```diff
@@ -3,6 +3,68 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        int ClientOS = 0;
+        if (ClientOS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+
+        int clientDeviceYear = 2015;
+        if (ClientOS == 0 && clientDeviceYear < 2015) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        } else {
+            if (ClientOS == 1 && clientDeviceYear < 2015) {
+                System.out.println("Установите облегченную версию приложения для Android по ссылке");
+            } else {
+                System.out.println("Установите приложение по ссылке");
+            }
+        }
+
+        int year = 2022;
+        if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
+            System.out.println(year + " год является високосным!");
+        } else {
+            System.out.println(year + " год не является високосным!");
+        }
+
+        int deliveryDistance = 95;
+        if (deliveryDistance < 20) {
+            System.out.println("Потребуется дней: 1 день!");
+        } else {
+            if (deliveryDistance >= 20 && deliveryDistance < 60) {
+                System.out.println("Потребуется дней: 2 дня!");
+            } else {
+                if (deliveryDistance >= 60 && deliveryDistance <= 100) {
+                    System.out.println("Потребуется дней: 3 дня!");
```

Comment:
начальные интервалы можно исключить поскольку они заведомо true из-за предыдущих условий
```suggestion
            if (deliveryDistance < 60) {
                System.out.println("Потребуется дней: 2 дня!");
            } else {
                if (deliveryDistance <= 100) {
                    System.out.println("Потребуется дней: 3 дня!");
```